### PR TITLE
Correct CASA installation instructions

### PR DESCRIPTION
**Pull request by keflavich**
This will be a fix to #7818.  I'd like to test it a little further before merging, though, so do not merge this yet.

Note that I added the CASA-Installation Issue label to help users find CASA-related issues in the astropy issue tracker.

### CODE REVIEW

**Comment by keflavich** [comment]
Actually, on looking a little closer, I think we can merge this if others approve it.  I believe there is still some sort of weird outstanding issue where in _some_ installs, ipython breaks horribly and won't run until you uninstall and reinstall `shutil_get_terminal_size`, but I can't verify that at the moment.  If it proves to be a real issue, we'll add the instructions laid out in [this issue](https://github.com/radio-astro-tools/casa-python/issues/19) and call it done.

**Comment by bsipocz** [comment]
I'm 👍 to merge this and add to LTS. 

However the label is a bit too specific, there won't be that many of these that one cannot navigate using the title of the issues.

File: docs/install.rst [hunk 1]
```diff
@@ -358,23 +355,24 @@ installed. Start up CASA as normal, and type::
...
-          pre-releases of CASA 4.7. However, due to missing header files in
-          CASA, they are known to **not** work on Linux with CASA 4.2.1 and
-          CASA 4.6.0.
+packages that may be useful for radio astronomy).
+
+.. note:: The above instructions have not been tested on all systems.
+   We know of a few examples that do work, but that is not a guarantee
+   that this will work on all systems.  If you install astropy and begin to
+   encounter issues with CASA, please look at the `known CASA issues
+   <https://github.com/astropy/astropy/issues?q=+label%3ACASA-Installation+>`_
```
Comment (bsipocz) [inline docs/install.rst]: https://github.com/astropy/astropy/search?q=casa+installation&type=Issues
```suggestion
   <https://github.com/astropy/astropy/issues?q=+label%3ACASA-Installation+>`_
```

Comment (bsipocz) [inline docs/install.rst]:
```suggestion
   <https://github.com/astropy/astropy/search?q=casa+installation&type=Issues>`_
```

**Comment by keflavich** [comment]
https://github.com/astropy/astropy/issues?utf8=%E2%9C%93&q=label%3Acasa-installation has 6 hits, https://github.com/astropy/astropy/search?q=casa+installation&type=Issues has only 4 and I think is the wrong search anyway; I want users searching in the issue pages directly.  

Is there a reason _not_ to have this label?

**Comment by bsipocz** [comment]
OK, then this is in the issues page:
https://github.com/astropy/astropy/issues?utf8=%E2%9C%93&q=casa+installation

> Is there a reason not to have this label?

I just feel it's way too upstream and specific for something non astropy, we don't have labels for way more related stuff like "pytest issues". 

At least we need to change the colour to be something very light and not red.

**Comment by keflavich** [comment]
OK, changing the color is totally reasonable.  More generally, we should talk about this at the f2f meeting.

**Pull request merged**